packages/shared/lib/services/connection.service.integration.test.ts

```diff
@@ -209,4 +210,47 @@ describe('Connection service integration tests', () => {
             expect(connectionIds).toEqual([notionOK.connection_id]);
         });
     });
+
+    describe('count', () => {
+        it('return total and error counts', async () => {
+            const env = await createEnvironmentSeed();
+
+            const config = await createConfigSeed(env, 'notion', 'notion');
+
+            await createConnectionSeed(env, 'notion');
+
+            const notionAuthError = await createConnectionSeed(env, 'notion');
+            await errorNotificationService.auth.create({
+                type: 'auth',
+                action: 'connection_test',
+                connection_id: notionAuthError.id!,
+                log_id: Math.random().toString(36).substring(7),
+                active: true
+            });
+
+            const notionSyncError = await createConnectionSeed(env, 'notion');
+            const sync = await createSyncSeeds({
+                connectionId: notionSyncError.id!,
+                environment_id: env.id,
+                nango_config_id: config.id!,
+                sync_name: 'test'
+            });
+            await errorNotificationService.sync.create({
+                type: 'sync',
+                action: 'sync_test',
+                connection_id: notionSyncError.id!,
+                log_id: Math.random().toString(36).substring(7),
+                active: true,
+                sync_id: sync.sync.id
+            });
+
+            const countResult = await connectionService.count({ environmentId: env.id });
+            expect(countResult.isOk()).toBeTruthy();
```

Comment:
you don't really need this one because if it is a error the `unwrap` below will thow

Reply:
Fixed in 49fefb91